packages/connectors/src/apps/discord/auth.ts

```diff
@@ -0,0 +1,107 @@
+import { addSeconds } from "date-fns";
+
+import { expiresIn5Minutes } from "@rollout/connectors/utils/expires-in-5mins";
+
+import { envSchema } from "./env";
+import assert from "assert";
+import { getUser } from "./lib/get-user"
+
+import { AppCredential, defineOAuth2 } from "@rollout/framework";
+
+export type DiscordCredential = AppCredential<{
+  accessToken: string;
+  refreshToken: string;
+  expireDate: string;
+  guildId: string;
+  botToken: string;
+}>;
+
+export const auth = defineOAuth2<DiscordCredential>()({
+  supportedByRolloutCredentialsService: true,
```

Comment:
```suggestion
  supportedByRolloutCredentialsService: true
```
 
The Rollout credentials service does not currently support Discord, it's whitelabel only

Reply:
done. Testing locally with `false` is not possible.